wpilib_interface/build.gradle

```diff
@@ -66,38 +125,35 @@ test {
 	systemProperty 'junit.jupiter.extensions.autodetection.enabled', 'true'
 }
 
-// Configure string concat to always inline compile
-tasks.withType(JavaCompile) {
-	options.compilerArgs.add '-XDstringConcat=inline'
+// Simulation configuration (e.g. environment variables).
+//
+// The sim GUI is *disabled* by default to support running
+// AdvantageKit log replay from the command line. Set the
+// value to "true" to enable the sim GUI by default (this
+// is the standard WPILib behavior).
+wpi.sim.addGui().defaultEnabled = false
+wpi.sim.addDriverstation()
+
+// Setting up my Jar File. In this case, adding all libraries into the main jar ('fat jar')
+// in order to make them all available at runtime. Also adding the manifest so WPILib
+// knows where to look for our Robot Class.
+jar {
+	from { configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
+	from sourceSets.main.allSource
+	manifest edu.wpi.first.gradlerio.GradleRIOPlugin.javaManifest(ROBOT_MAIN_CLASS)
+	duplicatesStrategy = DuplicatesStrategy.INCLUDE
 }
```

Comment:
This is extremely sus to me - I think this addition is probably causing the dependency issues in CI/locally since the jar is being build out of process (the jar should be handled by mavenPublisher rather than here)

Reply:
it was